a3p-integration/proposals/a:upgrade-next/priceFeed-follower-auction.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava';
+import { getDetailsMatchingVats } from './vatDetails.js';
+
+test('new auction vat', async t => {
+  const details = await getDetailsMatchingVats('auctioneer');
+  // This query matches both the auction and its governor
+  t.true(Object.keys(details).length > 2);
```

Comment:
this seems very loose. I don't know exactly what it should be checking but consider a `t.like` or at least a known length

Reply:
done